src/lib/Behat/BusinessContext/AdministrationContext.php

```diff
@@ -98,10 +96,8 @@ public function isElementOnTheList(string $listElementName, string $page, ?strin
      */
     public function isElementNotOnTheList(string $listElementName, string $page, string $parameter = null): void
     {
-        $isElementOnTheList = PageObjectFactory::createPage($this->utilityContext, $page, $parameter)
-            ->adminList->table->isElementInTable($listElementName);
-
-        if ($isElementOnTheList) {
+        $pageElement = PageObjectFactory::createPage($this->utilityContext, $page, $parameter);
+        if ($this->findElementOnTheList($pageElement, $listElementName)) {
```

Comment:
Why is this function in Context and not in AdminList/Table?

Reply:
Yeah, you are right